hack/overlays/set-envoy-daemonset-resources-limits.yaml

```
@@ -0,0 +1,28 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.all,expects="1+"
+#@overlay/match-child-defaults when=1
```

```suggestion
#@overlay/match by=overlay.subset({"kind":"DaemonSet"}),expects="1+"
```

We need the subset since the spec below only applies to DaemonSets. 

Also I think the `when=1` is hiding the fact that we aren't matching for some reason so the overlay isn't applied. Typically we should see the result of these overlays in `config/contour/external.yaml` and `config/contour/internal.yaml` after running `./hack/update-deps.sh`

---

Ah...... thanks for pointing this out @dprotaso. I fixed it, now it has applied properly and the config files have the resources added to them.